app/src/main/java/com/example/mycalculator/mvp/Operand.java

```diff
@@ -0,0 +1,36 @@
+package com.example.mycalculator.mvp;
```

Comment:
this class should not be here in this package

Reply:
Where do you recommend i should put it?

Reply:
in a utils package or in an entity package outside the mvp